features/step_definitions/buyers_management/account_steps.rb

```diff
@@ -56,13 +60,15 @@
 end
 
 When /^I check the buyers:$/ do |table|
+  table = transform_table(table)
   table.hashes.each do |hash|
     buyer = Account.buyers.find_by_org_name hash['buyer']
     check "account_bulk_#{buyer.id}"
   end
 end
 
 Then /^I should see the following buyers listed:$/ do |table|
+  table = transform_table(table)
```

Comment:
btw I has a script somewhere to add table transforms in another project migration :) Not sure how you did.